content/en/logs/log_configuration/archives.md

```diff
@@ -153,7 +153,18 @@ Add the following two permission statements to your IAM policies attached to the
 }
 ```
 
-[1]: /logs/archives/rehydrating/
+#### Attach the policy to the Datadog integration role
+ 
+Attach the new policy to the Datadog integration role.
+ 
+1. Navigate to **Roles** in the AWS IAM console.
+2. Locate the role used by the Datadog integration. By default it is named **DatadogIntegrationRole**, but the name may vary if your organization has renamed it. Click the role name to open the role summary page.
+3. Click **Add permissions**, and then **Attach policies**.
+4. Enter the name of the policy created above.
+5. Click **Attach policies**.
```

Comment:
```suggestion
   a. Navigate to **Roles** in the AWS IAM console.
   b. Locate the role used by the Datadog integration. By default it is named **DatadogIntegrationRole**, but the name may vary if your organization has renamed it. Click the role name to open the role summary page.
   c. Click **Add permissions**, and then **Attach policies**.
   d. Enter the name of the policy created above.
   e. Click **Attach policies**.

3. Edit the bucket names.
4. Optionally, specify the paths that contain your log archives.
```

Reply:
@kayayarai thank you for this feedback. Within this context, it doesn't make sense to edit the bucket names or specify the log archive paths after the policy is attached, so I adjusted the order of these steps.